### PR TITLE
Add swift driver

### PR DESCRIPTION
**Pull request by sileht**
This change adds the swift driver with auth 1.0

### CODE REVIEW

File: pifpaf/drivers/templates/swift/common.conf.inc [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+disable_fallocate = true
+workers = 1
+bind_ip = 127.0.0.1
+user = {{ USER }}
+swift_dir = {{ TMP_DIR }}
+devices = {{ TMP_DIR }}/data
+recon_cache_path =  {{ TMP_DIR }}
+log_address = {{ TMP_DIR }}/log
+log_level = DEBUG
+
```
Comment (jd) [inline pifpaf/drivers/templates/swift/common.conf.inc]: empty lines? :)

File: pifpaf/drivers/gnocchi.py [hunk 2]
```diff
@@ -90,16 +90,29 @@ def _setUp(self):
 
         if storage_driver == "s3":
             storage_config = {
-                "s3_access_key_id": (storage_parsed.username
+                "s3_access_key_id": (urlparse.unquote(storage_parsed.username)
```
Comment (jd) [inline pifpaf/drivers/gnocchi.py]: this does not seem related?

**Comment by jd** [comment]
Can you add the Fixes #45 in the commit message? other than that i'm good with it :)

**Comment by sileht** [comment]
I have fixed all remarks
